actors/market/src/types.rs

```diff
@@ -175,14 +175,15 @@ pub struct GetDealLabelReturn {
 pub type GetDealTermParams = DealQueryParams;
 #[derive(Serialize_tuple, Deserialize_tuple, Debug, Clone, Eq, PartialEq)]
 pub struct GetDealTermReturn {
-    pub start: ChainEpoch, // First epoch for the deal (inclusive)
-    pub end: ChainEpoch,   // Epoch at which the deal expires (i.e. exclusive).
+    pub start: ChainEpoch,    // First epoch for the deal (inclusive)
+    pub duration: ChainEpoch, // Duration of the deal.
 }
 
-pub type GetDealEpochPriceParams = DealQueryParams;
+pub type GetDealTotalPriceParams = DealQueryParams;
 #[derive(Serialize_tuple, Deserialize_tuple, Debug, Clone, Eq, PartialEq)]
-pub struct GetDealEpochPriceReturn {
-    pub price_per_epoch: TokenAmount,
+#[serde(transparent)]
```

Comment:
The `serde(transparent)` directive was missing from the old return, apologies for not catching in review.